DependencyInjection/SWPBridgeExtension.php

```diff
@@ -31,16 +31,37 @@ class SWPBridgeExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container)
     {
+        $defaultOptions = array();
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        if (!empty($config['base_uri'])) {
-            $container->setParameter($this->getAlias().'.base_uri', $config['base_uri']);
+        if (isset($config['api'])) {
+            if (!empty($config['api']['host'])) {
```

Comment:
you can repalce those duplicated if statements with foreach loop. Something like:

```php
$keys = array('host', 'port', 'protocol');
foreach($keys as $value) {
    if (!empty($config['api'][$value])) {
         $container->setParameter($this->getAlias() . '.api.'.$value, $config['api'][$value]);
    }
} 
```